ksql-rest-model/src/main/java/io/confluent/ksql/rest/entity/ActiveStandbyEntity.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest.entity;
+
+import static java.util.Objects.requireNonNull;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableSet;
+import java.util.Objects;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ActiveStandbyEntity {
+
+  private final Set<String> activeStores;
+  private final Set<String> activePartitions;
```

Comment:
this is actually storing topic-partitions, right?  In the form `<topic-name>-<partition-number> and its relying on `TopicPartition.toString()` to build that string, which we don't control. 

Much better to have `Set<TopicPartitionEntity>` so we / others don't need to parse the string and we're decoupled from any changes to `TopicPartition`.

Reply:
Done